.github/workflows/deploy_external.yaml

```diff
@@ -0,0 +1,23 @@
+# workflow containing webhooks that are external to us
+name: deploy_external
+on:
+  workflow_run:
+    workflows:
+      - "product_builder"
+    branches:
+      - "develop"
+    types:
+      - completed
+permissions:
+  contents: read
+  pull-requests: read
+  actions: read
+jobs:
+  execute-webhook:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Webhook
```

Comment:
What if we have 3 or more external instances? What will be their names? I think we can include the name of the company or use some type of code to avoid exposing any sensitive data.

Reply:
Modified as discussed.